playbook/app/pb_kits/playbook/pb_popover/popover.rb

```diff
@@ -53,6 +53,14 @@ def data
           pb_popover_close_on_click: close_on_click
         )
       end
+
+      def angle_down_path
```

Comment:
Just want to check- what does this do?

Do we need these methods for the angle icons?